template.json

```diff
@@ -0,0 +1,171 @@
+        "description": "The Azure SQL Server Administrator (SA) password"
+      }
+    },
+    "sqlServerActiveDirectoryAdminLogin": {
+      "type": "string",
+      "metadata": {
+        "description": "The active directory admin that will be assigned to the SQL server"
+      }
+    },
+    "sqlServerActiveDirectoryAdminObjectId": {
+      "type": "string",
+      "metadata": {
+        "description": "The object id of the active directory admin that will be assigned to the SQL server"
+      }
+    },
+    "virtualNetworkAddressPrefix": {
+      "type": "string",
+      "defaultValue": "10.0",
+      "metadata": {
+        "description": "First 2 octects of VNet Private IP address range (VNet prefix)"
+      }
+    },
+    "virtualNetworkRoutedSubnets": {
+      "type": "array",
+      "metadata": {
+        "description": "Array of subnets to create"
+      }
+    },
+    "virtualNetworkNonRoutedSubnets": {
+      "type": "array",
+      "metadata": {
+        "description": "Array of subnets to create"
+      }
+    },
+    "deployManageSql": {
+      "type": "bool",
+      "metadata": {
+        "description": ""
+      }
+    },
+    "msqlVNetResourceGroupName": {
+      "type": "string",
+      "metadata": {
+        "description": "The resourcee group where Managaged SQL in located"
+      }
+    }
+  },
+  "variables": {
+    "dfcDevOpsTemplatesUrl": "https://raw.githubusercontent.com/SkillsFundingAgency/dfc-devops/master/ArmTemplates/",
+    "localTemplatesUrl": "https://raw.githubusercontent.com/SkillsFundingAgency/dfc-shared/master/Resources/",
+    "msqlServerName": "[concat(parameters('sharedPrefix'),'-msql')]",
+    "managedSQLNSG": "[concat(parameters('sharedPrefix'),'-nsg')]",
+    "msqlSubnetName": "[concat(parameters('sharedPrefix'),'-sn')]",
+    "msqlVnetName": "[concat(parameters('sharedPrefix'),'-vnet')]"
+  },
+  "resources": [
+    {
+      "name": "sharedManagedInstanceEnvironment",
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2017-05-10",
+      "condition": "[equals(parameters('envType'),'dev')]",
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[concat(variables('dfcDevOpsTemplatesUrl'),'network.json')]",
+          "contentVersion": "1.0.0.0"
+        },
+        "parameters": {
+          "virtualNetworkPrefix": {
+            "value": "[parameters('sharedPrefix')]"
+          },
+          "virtualNetworkAddressPrefix": {
+            "value": "[parameters('virtualNetworkAddressPrefix')]"
+          },
+          "virtualNetworkRoutedSubnets": {
+            "value": "[parameters('virtualNetworkRoutedSubnets')]"
+          },
+          "virtualNetworkNonRoutedSubnets": {
+            "value": "[parameters('virtualNetworkNonRoutedSubnets')]"
+          }
+        }
+      }
+    },
+    {
+      "name": "managedSQLNSG",
+      "type": "Microsoft.Resources/deployments",
+      "condition": "[parameters('deployManageSql')]",

```

Comment:
Is there ever a case where you are going to run this and not want to deploy the managed SQL? Assuming not then you can completely remove the deployManageSql param

Reply:
Maybe, if the network needs to change.